Tasks/PublishSymbols/LegacyPublishHelpers.ps1

```diff
@@ -0,0 +1,219 @@
+########################################
+# (Script-scope variables) Derived from PublishHelpers\SemaphoreFunctions.ps1.
+########################################
+$script:maximumSemaphoreAge = [timespan]::FromDays(1)
+$script:sleepInterval = [timespan]::FromSeconds(10)
+
+########################################
+# Derived from PublishHelpers\CommonFunctions.ps1
+########################################
+function Get-SymStorePath {
+    $symstorePath = Get-ToolPath -Name 'Symstore\symstore.exe'
+    $null = Get-Item -LiteralPath $symstorePath -ErrorAction Stop
+    $symstorePath
+}
+
+function Get-ValidValue {
+    [CmdletBinding()]
+    param(
+        [timespan]$Current,
+        [timespan]$Minimum,
+        [timespan]$Maximum)
+
+    Write-Verbose "Entering Get-ValidValue"
+    Write-Verbose " Current: '$Current'"
+    Write-Verbose " Minimum '$Minimum'"
+    Write-Verbose " Maximum: '$Maximum'"
+    try {
+        if ($Current -lt $Minimum) { return $Minimum }
+        elseif ($Current -gt $Maximum) { return $Maximum }
+        else { return $Current }
+    } finally {
+        Write-Verbose "Leaving Get-ValidValue"
+    }
+}
+
+########################################
```

Comment:
We might not need unpublish for legacy publish symbol tasks. since the clean up job will only be queued after DT's hosted upgrade finish.